.github/actions/docker/action.yaml

```diff
@@ -41,7 +41,7 @@ runs:
     # to use tag generated by the image build.
     - name: Set Docker Tag
       id: set-docker-tag-presubmit-non-fork
-      if: ${{ (steps.changed-files.outputs.any_changed == 'false') && (github.event_name == 'pull_request') && (!github.event.pull_request.head.repo.fork) }}
+      if: ${{ (steps.changed-files.outputs.any_changed == 'false') && (github.event_name == 'pull_request') }}
```

Comment:
We want to do this step _and_ the step below in forked repos?

Reply:
This particular step should run on ALL pull requests (forked and non forked) when no docker related files were changed; effectively, use an existing docker image from the base branch (e.g. main).  

The step below should only be executed on forked PRs when change touches docker related file(s) (requires re-build of docker image and push to GCR).

Reply:
I see, now I see that the below step won't run in the same build as this step because of the changed-files condition. This LGTM.